libcxx/test/std/algorithms/alg.modifying.operations/alg.partitions/stable_partition.pass.cpp

```diff
@@ -282,10 +277,10 @@ test()
     assert(array[9] == P(0, 2));
   }
 #if TEST_STD_VER >= 11 && !defined(TEST_HAS_NO_EXCEPTIONS)
-  // TODO: Re-enable this test once we get recursive inlining fixed.
+  // TODO: Re-enable this test for GCC once we get recursive inlining fixed.
```

Comment:
Do you know whether there is a GCC bug report? If there is it would be great to add a link here.

Reply:
It seems that this is not a GCC bug. When compiling with GCC, our `_LIBCPP_HIDE_FROM_ABI` contains `__attribute__((__always_inline__))`, but many of our implementation details are recursive, and thus possibly trigger errors.

https://reviews.llvm.org/D154161 is related.